src/api/spec/models/staging/workflow_spec.rb

```diff
@@ -20,6 +20,11 @@
     staging_workflow
   end
 
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:managers_group) }
+    it { is_expected.to validate_presence_of(:project) }
```

Comment:
We usually don't test the (presence of) validations. Only if they contain logic.

Reply:
hm, i like the tests to make sure that the validation exists and wasn't disabled (i.e some disable it to debug something else, and forget to enable it back) or removed by mistake.

Reply:
If it's not possible to create a staging workflow w/o associated project (via the UI) there might be no other way to test the validation.
Plus these tests are quite cheap, no need to hit the db either.